src/components/Card.js

```diff
@@ -1,15 +1,28 @@
-import { eventType, selectors } from '../constants/app-data.js';
+import {
+	BASE_URL,
+	HEADERS,
+	eventType,
+	selectors,
+} from '../constants/app-data.js';
+import Api from './Api.js';
+import PopupDelete from './PopupDelete.js';
```

Comment:
From the 8th sprint it's not allowed to import anything to the classes. Only class `Popup` can be imported for extending. **`Loose coupling`** should be used: it means that you need to pass callbacks to the constructor rather than using outside elements or functions.